platform/web/request.go

```diff
@@ -78,7 +78,13 @@ func DecodeWithEmbeddedJSONSchema(request *http.Request, model interface{}, json
 }
 
 func validateRequestPayload(request *http.Request, model interface{}, options DecoderOptions, jsonSchema gojsonschema.JSONLoader) error {
-	body, _ := io.ReadAll(request.Body)
+	body, err := io.ReadAll(request.Body)
+	if err != nil {
+		if err.Error() == "http: request body too large" {
```

Comment:
where does this string come from actually ?

Reply:
It's the error returned by the `http.MaxBytesReader`, sadly they don't return an exported error as they usually do, though there's a proposal for it: https://github.com/golang/go/issues/41493